Packs/EWS/.pack-ignore

```diff
@@ -15,3 +15,7 @@ ignore=RM102,RM106
 
 [file:classifier-mapper-incoming-EWS_v2.json]
 ignore=BA101
+
+[known_words]
+***!O365-sc-auth-start***
```

Comment:
Why do you need it?

Reply:
The `demisto-sdk doc review` failed locally. I didn't check remotely